src/components/FormComponent/FormComponent.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import InputFieldComponent from '../InputFieldComponent/InputFieldComponent';
+import "./FormComponent.css";
+
+class FormComponent extends Component {
+	constructor() {
```

Comment:
Indentation is quite off here. As @kim-kolosek  said, try using two spaces for indentation - I'll configure linter to force this